### DISCUSSION TITLE
違うキーとして反応（認識）する

### DISCUSSION THREAD

**Discussion by tononeco** [Yowkees/keyball#229] [Q&A] [answered]
Keyball 61を組み立てましたところ一箇所だけ違うキーとして反応するキーがあります。
写真の○印のキーを押下すると□印のキーとして反応します。
![不具合箇所](https://user-images.githubusercontent.com/132801653/236658054-e5fa9cc7-c060-4195-8395-a03b9f22e6e7.jpg)
REMAPのTest Matrix Modeでも同様に○印のキーを押下すると□印のキーとして反応します。
OLEDの表示はどちらのキーを押しても以下の写真の表示となります。
![OLED表示](https://user-images.githubusercontent.com/132801653/236658306-3694cf94-5af4-4f8a-84c5-d822d7c1ac30.jpg)

お手数ですが解決方法をご教示いただけますと幸いです。

**Comment by Yowkees** [comment, marked as answer]
ご連絡が遅くなり申し訳ありませんでした。
問題が起きているキー周辺のダイオードの向きが間違っている可能性が高いです。こちらを再度ご確認お願いします。

下記にKeyball61の基板データを公開していますので、KiCAD等フリーソフトで表示いただいて線を追っていただくと、どのダイオードが怪しいか特定できるかと思います。
https://github.com/Yowkees/keyball/tree/main/keyball61/design_data

半田付け面のお写真を見せていただければ何かわかるかも知れません。
お手数ですがご対応よろしくお願いいたします。

**Reply by tononeco** [reply]
ご回答いただきありがとうございます。
ご教示いただいた通り周辺のダイオードを確認した所、一箇所向きを誤って取り付けておりました。
取り付け直しましたら無事正常な動作となりましたのでご報告させていただきます。
お忙しい中ご対応いただきありがとうございました。
楽しく作成でき使い心地の良い素晴らしいキーボードを開発いただき感謝しております。

**Comment by Yowkees** [comment]
無事に直ったようで良かったです！安心しました。
ぜひ末永くご愛用いただければ幸いです。
